test/e2e.js

```diff
@@ -328,7 +337,7 @@ describe('plotly database connector', function Spec() {
 
                 // click on the evaluated dialect logo
                 this.fillInputs(testedDialect)
-                .then(await delay(500))
+                .then(await delay(1000))
```

Comment:
these delays are odd, I would've expected that if `fillInputs(testedDialect)` would return a promise when it was totally done